src/crypto/x509/pkix/pkix_test.go

```diff
@@ -0,0 +1,24 @@
+package pkix
+
+import (
+	"testing"
+)
+
+func TestEmailAddress(t *testing.T) {
+	name := &Name{
+		Country:            []string{"US"},
+		Organization:       []string{"Hot Wheels LLC"},
+		OrganizationalUnit: []string{"Sweet Rides"},
+		Locality:           []string{"San Francisco"},
+		Province:           []string{"California"},
+		CommonName:         "test.com",
+		EmailAddress:       "hot@wheels.com",
```

Comment:
In the rare case that the owner of this domain complains, can we use something such as `example.com` and `user@example.net`?

Reply:
Done.